examples/experimental/persistent-volume-provisioning/README.md

```diff
@@ -81,6 +81,28 @@ parameters:
 * `type`: `pd-standard` or `pd-ssd`. Default: `pd-ssd`
 * `zone`: GCE zone. If not specified, a random zone in the same region as controller-manager will be chosen.
 
+#### GLUSTERFS
+
+```yaml
+apiVersion: extensions/v1beta1
+kind: StorageClass
+metadata:
+  name: slow
```

Comment:
Thats the standard name we use in kubernetes. Yes, this is an example and we are mentioning a common name for the same.